docs/topics/multiplatform/multiplatform-direct-integration.md

```diff
@@ -0,0 +1,89 @@
+[//]: # (title: Direct integration)
+
+If you already have a Kotlin Multiplatform and an iOS project and want to develop them simultaneously to share code, you
+can set up direct integration through a special `embedAndSignAppleFrameworkForXcode` script.
+
+This script automates the process of connecting the Kotlin framework to iOS projects in Xcode. It's designed
+specifically for the Xcode environment.
+
+![Direct integration diagram](direct-integration-scheme.svg){width="700"}
+
+The idea of direct integration is to make the Kotlin build a part of the iOS build.
+The script is added to the run script phase of the iOS app build so that the Kotlin artifact is built and included in
+the derived data before running the iOS app build.
+
+In general, the script:
+
+* Copies the compiled Kotlin framework into the correct directory within the iOS project structure.
+* Handles the code signing process of the embedded framework.
+* Ensures that code changes in the Kotlin framework are reflected in the iOS app in Xcode.
+
+## How to set up
+
+If you're currently using the CocoaPods plugin to connect your Kotlin framework, migrate first.
+
+### Migrate from CocoaPods plugin to direct integration {initial-collapse-state="collapsed"}
+
+To migrate from the CocoaPods plugin:
+
+1. In Xcode, clean build directories using **Product** | **Clean Build Folder** or with the
+   <shortcut>Cmd + Shift + K</shortcut> shortcut.
+2. In the directory with `Podfile`, run the following command:
+
+    ```none
+   pod deintegrate
+   ```
+
+3. Remove the `cocoapods {}` block from your `build.gradle(.kts)` files.
+4. Delete the `.podspec` and `Podfile` files.
+
+### Connect the framework to your project
+
+To connect the Kotlin framework generated from the multiplatform project to your Xcode project:
+
+1. The `embedAndSignAppleFrameworkForXcode` task only registers if the `binaries.framework` configuration option is
+   declared. In your Kotlin Multiplatform project, check the iOS target declaration in the `build.gradle.kts` file.
+2. In Xcode, open the iOS project settings by double-clicking the project name.
+3. On the **Build Phases** tab of the project settings, click the **+** and add **New Run Script Phase**.
+
+   ![Add run script phase](xcode-run-script-phase-1.png){width=700}
+
+4. Adjust the following script and add it in the run script phase:
```

Comment:
```suggestion
4. Adjust the following script and copy the result to the run script phase:
```

For me the natural order is "copy to Xcode, then edit there", but I understand that it's good to put "adjust" in the beginning so that people don't forget to do that.